virttest/env_process.py

```diff
@@ -1552,6 +1568,18 @@ def postprocess_on_error(test, params, env):
     params.update(params.object_params("on_error"))
 
 
+def _get_screendump_format(params):
+    image_format = "ppm"
+
+    if (
+        params.get("vm_type") == "libvirt"
+        and utils_misc.compare_qemu_version(7, 1, 0, False, params)
+        and libvirt_version.version_compare(9, 0, 0)
```

Comment:
Please add the comments about the background in order to let the reviewer get more details. Thanks.

Reply:
Got it, the background comment will be added in the next update.